packages/cli/test/commands/deploy.test.ts

```diff
@@ -41,6 +41,29 @@ describe('deploy', () => {
       })
     })
 
+    fancy.stdout().it('calls pruneDependencies function', async (ctx) => {
+      const fakeProvider = {} as ProviderLibrary
+
+      const fakeLoader = fake.resolves({
+        provider: fakeProvider,
+        appName: 'fake app',
+        region: 'tunte',
+        entities: {},
+      })
+
+      const fakeDeployer = fake((_config: unknown, logger: Logger) => {
+        logger.info('this is a progress update')
+      })
+
+      const pruneDependenciesSpy = spy(deploy, deploy.pruneDependencies)
```

Comment:
I believe this should be
```suggestion
      const pruneDependenciesSpy = spy(deploy, 'pruneDependencies')
```